packages/sdk-browser/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/sdk-browser
 
+## 1.0.0
```

Comment:
should we make note that this is a stable release, which warrants a jump from 0.X.X to 1.X.X?

Reply:
and same goes for sdk-browser / sdk-react